framework/wazuh/agent.py

```diff
@@ -1966,16 +1966,22 @@ def _get_wpk_file(self, wpk_repo=common.wpk_repo_url, debug=False, version=None,
 
         # Comparing versions
         agent_ver = self.version
-        if debug:
-            print("Agent version: {0}".format(agent_ver.split(" ")[1]))
-            print("Agent new version: {0}".format(agent_new_ver))
+
 
         if WazuhVersion(manager_ver.split(" ")[1]) < WazuhVersion(agent_new_ver):
-            raise WazuhException(1717, "Manager: {0} / Agent: {1} -> {2}".format(manager_ver.split(" ")[1], agent_ver.split(" ")[1], agent_new_ver))
```

Comment:
This exception cannot be removed. The agent should not be upgraded to a higher version than the manager without including the force flag `-F`.